vmas/scenarios/navigation.py

```diff
@@ -283,6 +290,98 @@ def extra_render(self, env_index: int = 0) -> "List[Geom]":
         return geoms
 
 
+class HeuristicPolicy(BaseHeuristicPolicy):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.clf_epsilon = 0.2  # Exponential CLF convergence rate
+        self.cbf_eta = 1  # CBF class-K parameter
+        self.cbf_zeta = 1  # CBF class-K parameter
+        self.clf_slack = 100.0  # weights on CLF-QP slack variable
```

Comment:
This should be input params to the constructor with default values and not hardcoded

Reply:
The control parameters is defined in constructor.